docs/guides/TROUBLESHOOTING.md

```
@@ -791,46 +695,25 @@ Acceptance: 6 xfailed tests in test_validation.py pass, exit code 0
-   # Then 10 messages back and forth
-
-   # ✅ CHEAP: One complete message
-   aider --files src/validation.py \
-         --read plan.md \
-         --message "[Complete detailed instructions]" \
-         --yes
-   ```
-
-3. **Large files in context**
+1. **Large files in context**
    ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)